pilot/pkg/config/memory/store.go

```diff
@@ -161,12 +159,13 @@ func (cr *store) Create(cfg config.Config) (string, error) {
 	_, exists = ns.Load(cfg.Name)
 
 	if !exists {
-		tnow := time.Now()
-		cfg.ResourceVersion = tnow.String()
-
 		// Set the creation timestamp, if not provided.
 		if cfg.CreationTimestamp.IsZero() {
-			cfg.CreationTimestamp = tnow
+			cfg.CreationTimestamp = time.Now()
+		}
+
+		if cfg.ResourceVersion == "" {
+			cfg.ResourceVersion = time.Now().String()
```

Comment:
nit: may be best to use the same time for creation time and resource version. otherwise could be a bit confusing

Reply:
> nit: may be best to use the same time for creation time and resource version. otherwise could be a bit confusing

yes，thx